src/fu-device.c

```diff
@@ -1103,6 +1103,26 @@ fu_device_get_metadata_integer (FuDevice *self, const gchar *key)
 	return (guint) val;
 }
 
+/**
+ * fu_device_clear_metadata:
```

Comment:
Maybe 'remove'?  Do we have precident in the API?

Reply:
We don't have any previous instance of clearing - the set function actually failed if you set `NULL` for value.  I'll change to remove.